### ISSUE TITLE
Experiment with transifex for (some of?) our documentation

### ISSUE THREAD

**Issue by choldgraf** [jupyterhub/team-compass#311]
In https://github.com/jupyter/jupyter/pull/475, @parente graciously wrote down his knowledge about how to use transifex to get community support for translating our Sphinx documentation into other languages. It would be great if we could try and leverage these instructions to translate parts of the JupyterHub docs into other languages. This could potentially makes our documentation much more accessible to the non-english-speaking world.

Perhaps a good start would be to pick one repository that we think is particularly relevant to a broad audience, and set up the infrastructure for transifex following Peter's instructions? Maybe we can discuss at the next team meeting.

**Comment by minrk** [comment]
This sounds great! My first thoughts are:

- z2jh is probably the broadest reaching repo in terms of readers, or
- tljh, which is a similar repo, but has a smaller, possibly more tractable, scope as a first experiment, if that's relevant

jupyterhub/jupyterhub is the most 'foundational' docs we have, but given the existence of these two 'distribution' docs repos, I expect the base docs are mostly read for developer/advanced use cases in practice.

**Comment by sgibson91** [comment]
@tonyyzy has been doing a fabulous job deploying transifex over on the Turing Way and may have some tips for us

**Comment by choldgraf** [comment]
yes! it would be awesome to have some guidance on this. @tonyyzy would also love your thoughts on the workflow here: https://jupyter.readthedocs.io/en/latest/contrib_docs/doc-new-translation.html?highlight=transifex

and btw, it would be great if we could provide suggestions to people for using transifex with jupyter book!

@minrk I agree re: TLJH and Z2JH being the most-likely to be useful to a broad audience...maybe in particular TLJH?)

**Comment by tonyyzy** [comment]
Hi all! Thanks for the mention @sgibson91 
@choldgraf We heavily borrowed a lot of material from Jupyter, carpentries, and other projects. The flowchart looks amazing, we might steal it for our documentation ;)
The workflow is more or less the same, except we don't automatically generate the `.po` files yet. If the source po files get updated then the translated strings in Transifex need to be reviewed so we haven't paid much attention to keeping the source up-to-date (mainly my negligence). Another difference is we use [po4jupyter](https://github.com/tonyyzy/po4jupyterbook) which is a python wrapper around `gettext`, derived from [po4gitbook](https://github.com/darkcircle/po4gitbook) which carpentries uses.

We only have Markdown content at the moment so it's all well, it's still uncertain how well it will behave with notebook files.

**Comment by betatim** [comment]
Slightly off-topic: are there alternatives to transifex that are open-source/have a BSD style license?
